commands.go

```diff
@@ -136,8 +136,9 @@ func CmdResetTopics() ([]byte, error) {
 
 // CmdSetIDKey creates a command to set the client private key to the given key
 func CmdSetIDKey(key []byte) ([]byte, error) {
-	if keyLen := len(key); keyLen != e4crypto.KeyLen {
-		return nil, fmt.Errorf("invalid key length, got %d, wanted %d", keyLen, e4crypto.KeyLen)
+	keyLen := len(key)
+	if keyLen != e4crypto.KeyLen && keyLen != ed25519.PrivateKeySize {
+		return nil, fmt.Errorf("invalid key length, got %d, wanted %d or %d", keyLen, e4crypto.KeyLen, ed25519.PrivateKeySize)
```

Comment:
Same comment as above - I feel like we should know which size we want. Any ideas would be useful I think.

Reply:
I don't really see how to check for the proper size here, as it depends on the caller. To me it's the caller responsibility to create a command with the proper key size - we validate on sizes we support, but I'm not even sure we should. One may want to create a command with a 128 or 256 bytes key, which is fine if the client receiving this command accept it.